docs/develop/parent-to-dialog.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Passing data and messages to a dialog box from its host page
+description: 'Learn how to pass data to a dialog from the host page'
+ms.date: 03/11/2020
+localization_priority: Normal
+---
+
+# Passing data and messages to a dialog box from its host page
+
+Microsoft is now making available for preview new APIs for messaging from the host page (defined [here](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page)) to the dialog box.
+
+> [!Important]
+>
+> - The APIs are in preview. They are available to developers for experimentation; but they should not be used in a production add-in. Until this API is released, use the techniques described in [Pass information to the dialog box](dialog-api-in-office-add-ins.md#pass-information-to-the-dialog-box) in production add-ins.
+> - The APIs described in this article require Office 365 (the subscription version of Office). You should use the latest monthly version and build from the Insiders channel. You need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1). Please note that when a build graduates to the production semi-annual channel, support for preview features is turned off for that build.
+> - The APIs are only supported in Excel during the preview.
+
+## Use messageChild() from the host page
+
+When you call the Office dialog API to open a dialog box, a [Dialog](/javascript/api/office/office.dialog) object is returned and should be assigned to a variable which typically has greater scope than the [displayDialogAsync](/javascript/api/office/office.ui#displaydialogasync-startaddress--callback-)
+method because the object will be referenced in other methods. The following is an example:
+
+```javascript
+var dialog;
+Office.context.ui.displayDialogAsync('https://myDomain/myDialog.html',
+    function (asyncResult) {
+        dialog = asyncResult.value;
+        dialog.addEventHandler(Office.EventType.DialogMessageReceived, processMessage);
+    }
+);
+
+function processMessage(arg) {
+    dialog.close();
+
+  // message processing code goes here;
+
+}
+```
+
+This `Dialog` object has a `messageChild` method that will send any string, or stringified data, to the dialog box. This raises a `DialogParentMessageReceived` event in the dialog box. Your code should handle this event. (See next section.)
```

Comment:
```suggestion
This `Dialog` object has a `messageChild` method that sends any string, or stringified data, to the dialog box. This raises a `DialogParentMessageReceived` event in the dialog box. Your code should handle this event, as shown in the next section.
```

(I think it would also be valid to link to the next section, to make it very obvious to what you're referring.)